portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,34 +34,41 @@
 @WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
-    private List<Comment> commentsList = new ArrayList<Comment>(
-        List.of(new Comment("I really like your webpage!", new Date()))
-      );
+  DatastoreService datastore;
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String commentsInJson = convertToJsonUsingGson(commentsList);
+    Query query = new Query("Comment").addSort("dateCreated", SortDirection.DESCENDING);
+    datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Please initialize the datastore in init, so that you don't create a datastore every time